test/orbitals.jl

```diff
@@ -323,4 +323,82 @@ using Random
         @test hash(o"3s") == hash(o"3s")
         @test hash(ro"3p-") == hash(ro"3p-")
     end
+
+    @testset "Serialization" begin
+        @testset "Orbitals" begin
+            o = o"1s"
+            p = o"kg"
+            q = Orbital(:k̃, 14)
+            oo = SpinOrbital(o, (0, 1/2))
+            r = Orbital(Symbol("[$(oo)]"), 14)
+
+            no,np,nq,nr = mktempdir() do path
+                filename = joinpath(path, "orbitals")
+                open(filename, "w") do file
+                    foreach(Base.Fix1(write, file), (o,p,q,r))
+                end
```

Comment:
Could probably use `IOBuffer` instead for these tests?

Reply:
Of course it should, thanks.